src/ui/shared/BrutalistUI.jsx

```
@@ -0,0 +1,1291 @@
+}
+
+// 11. Void Loader (Geometric Spinner)
+export const VoidLoader = ({ size = 'w-16 h-16' }) => {
+  return (
+    <div className={`relative ${size} flex items-center justify-center`}>
+      {/* Outer Hex - Slow counter-clockwise */}
+      <svg
+        className='absolute inset-0 w-full h-full text-(--toxic-green) animate-[spin_4s_linear_infinite_reverse]'
+        viewBox='0 0 100 100'
+        fill='none'
+      >
```

Biome (2.4.4)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)